### PR TITLE
HPCC-27424 Document 64-bit Client Tools should be used for large WUs

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by GordonSmith — APPROVED** [review]
@JamesDeFabia looks ok - It might be worth highlighting that in general the user should always install the 64bit version of client tools (unless they have a 32bit windows, which is not likely)

**Comment by richardkchapman** [comment]
I think there are cases where the 64-bit compiler uses more memory, on weedy machines this can be a problem? @ghalliday can you comment?

**Comment by ghalliday** [comment]
Yes.  If your machine only has 4Gb of memory you are probably better off using the 32bit version of the client tools (I would estimate eclcc consumes up to 10% less memory).  If you have more than 4GB you are probably better off with the 64bit tools.
I don't know how the speed compares running on a 64bit OS.

**Comment by JamesDeFabia** [comment]
I updated based on Richard and Gavin's feedback. @GordonSmith please review again

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_Overview.xml [hunk 1]
```diff
@@ -150,9 +166,11 @@
       the package, use:</para>
 
       <programlisting>sudo dpkg -i &lt;deb filename&gt;</programlisting>
-	  <para>After installing the package, run the following to update any dependencies:</para>
 
-        <programlisting>sudo apt-get install -f </programlisting>
+      <para>After installing the package, run the following to update any
+      dependencies:</para>
```
Comment (GordonSmith) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_Overview.xml:171]: `run the following to "fix" any missing dependencies`

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Minor:  more truthful explanation of what `apt-get install -f` does

**Review by GordonSmith — CHANGES_REQUESTED** [review]
As per comments

**Review by GordonSmith — APPROVED** [review]

**Comment by JamesDeFabia** [comment]
Squashed. 
@richardkchapman This should be good to merge now

**Pull request merged**